libs/client/ui/pages/src/lib/exercise-detail/exercise-detail.tsx

```diff
@@ -1,15 +1,10 @@
 import './exercise-detail.scss';
```

Comment:
I find gad die stelle ned wo des beeinflusst aba da slider isch zmol viel witta domma isch des gewollt?
![image](https://user-images.githubusercontent.com/54234287/201535509-168fe7eb-e824-4e7b-9ca8-abfd59ea9a9c.png)

Reply:
😂 wo i des beeinfluss
Des was i to hob war halt das i die Toolbar ganz domma glöscht hob - und du hosch halt deam Flex-Container 100% height geaba, ergo er war davor immer größer als da display - ergo du hosch da margin-bottom vom Timer uf 80 px gsetzt so das ma ean wieder siat.
I hob des aber net so beachtet, weil i im ProductionContent Branch grad die exercise detail umbau und umstyle, weil do jo no die ganzen discription blöcke dazua kummen.